trie/triedb/pathdb/nodebufferlist.go

```diff
@@ -598,7 +806,9 @@ func (nf *nodebufferlist) loop() {
 				continue
 			}
 			nf.diffToBase()
+			log.Info("check base size and limit", "base size", nf.base.size, "limit", nf.base.limit)
```

Comment:
Is the log necessary? Will it print a lot?

Reply:
Deleted